internal/action/generate.go

```diff
@@ -111,7 +111,7 @@ func keyAndLength(args argList) (string, string) {
 func (s *Action) generateCopyOrPrint(ctx context.Context, c *cli.Context, name, key, password string) error {
 	entry := name
 	if key != "" {
-		entry += ":" + key
+		entry += " " + key
```

Comment:
This should have broken some test 😢
Why not keep the entry:key format?

Reply:
Because it does not work. IIRC we did never support that format.